plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -87,6 +87,7 @@
       "https://console.cloud.google.com/appengine/versions";
   private static final String CREATE_GCP_PROJECT_WITH_GAE_URL =
       "https://console.cloud.google.com/projectselector/appengine/create?lang=java";
+  private static final String UNICODE_REFRESH_SYMBOL = "\uD83D\uDD04";
```

Comment:
I remember our style guide instructs to write unicode symbols directly and not by codes. Let me look for it.